table-expressions.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Table Expressions
+summary: Table expressions define a data source in SELECT clauses and INSERT statements.
+toc: false
+---
+
+Table expressions define a data source in SELECT clauses and INSERT statements.
+
+<div id="toc"></div>
+
+## Introduction
+
+Table expressions are used prominently in the `SELECT` clause:
+
+	SELECT ... FROM <table expr>, <table expr>, ...
+
+CockroachDB recognizes the following table expressions:
+
+- a table or view name
+- a table generator function
+- a `SELECT` or `VALUES` clause, as a sub-query
+- an aliased table expression, using an `AS` clause
+- an explicit `JOIN` expression
+- another table expression annoted with `WITH ORDINALITY`
+- another table expression between parentheses,
+
+The following sections provide details on each of these options.
+
+In addition to this, the `FROM` clause itself accepts more than one
+consecutive table expressions at the top level, separated by
+commas. This is a shorthand notation for `CROSS JOIN`, documented in
+the `JOIN` syntax below.
+
+## Table or view names
```

Comment:
Same comment as in `sql-expressions.md`: Let's use title case for all headings, e.g., `Table or View Names`.